src/Backoffice/Employees/Domain/Actions/ListEmployeesAction.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employees\Domain\Actions;
+
+use Illuminate\Database\Eloquent\Collection;
+use Lightit\Backoffice\Employees\Domain\Models\Employee;
+
+class ListEmployeesAction
+{
+    /**
+     * @return Collection<int, Employee>
+     */
+    public function __invoke(): Collection
```

Comment:
```suggestion
    public function execute(): Collection
```
This is the standard here at Light-it. But I'm unsure if you went with an invokable action because of other guidelines. Just letting you know about what the standard we follow it.

Reply:
I agree with @gianfranco-rocco. Also, why is there a difference between the definition of an action in the Users module and here? I'm not saying one option is better than the other, just that we should keep consistency throughout the project